README.md

```diff
@@ -21,6 +21,7 @@
 
 ### Native
 
+0. If you haven't already, follow this [tutorial](https://gorails.com/setup) to setup your Rails environment, make sure to install PostgreSQL 9.6 as the database
```

Comment:
Starting at zero is such a linux/stallman thing to do :laughing:

Reply:
Also today I learned some html

![Selection_707](https://user-images.githubusercontent.com/19378/61451472-5c730e00-a951-11e9-8b50-d60a1737b764.png)